src/app.rs

```diff
@@ -28,7 +28,10 @@ pub(crate) fn run(app_config: AppConfig) -> Result<()> {
 
     // 3. Build the specified target
     let cargo_config = Config::default()?;
-    let manifest_path = important_paths::find_root_manifest_for_wd(cargo_config.cwd())?;
+    let manifest_path = app_config
+        .manifest_path
+        .clone()
+        .map_or_else(|| important_paths::find_root_manifest_for_wd(cargo_config.cwd()), Ok)?;
```

Comment:
I think we can make this worth with relative paths via something like:

```suggestion
    let manifest_path = match app_config.manifest_path.as_ref() {
        Some(path) if path.is_absolute() => Ok(path.to_owned()),
        Some(path) => Ok(cargo_config.cwd().join(path)),
        None => important_paths::find_root_manifest_for_wd(cargo_config.cwd()),
    }?;
```

Unless there is some reason we might not want to do this? It seems like a clear improvement to ergonomics, I was confused when I tried to test this and couldn't pass a relative path. 🤷